tools/sof-get-kernel-line.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
```

Comment:
I will suggest to have some inline description with example output here.

You do not only output line number but also kernel version info.
```
22 4.15.0-74-generic
1092 4.15.0-74-generic
2156 4.15.0-74-generic
3246 4.15.0-74-generic
```